test/SlowTests.hs

```diff
@@ -29,7 +29,7 @@ main = defaultMain suite
 
 suite :: TestTree
 suite = testGroup "ChainwebSlowTests"
-    [ Chainweb.Test.MultiNode.test Warn (TestWithTime petersonChainGraph) 10 120 Nothing
+    [ Chainweb.Test.MultiNode.test Warn (TestWithTime petersonChainGraph) 10 300 Nothing
```

Comment:
Let's try and hope it's not flaky and too annoying.

If it turns out to be too long or flaky, we can just lower thresholds radically and run only for, say, 60 or 120 seconds.

Reply:
Eventually, we should also make the time and thresholds configurable. Unfortunately that's a bit tricky with tasty. But we could always create a separate non-tasty version of it or just drop the tasty version.

Reply:
:+1:  I'll give this a try and we can re-adjust later.